web-app/src/main/webapp/css/style.css

```diff
@@ -23,3 +24,28 @@
   font-size: 2em;
   height: 3em;
 }
+
+/* Report form */
+#report-form {
+  position: absolute;
+  top: 200px;
+  z-index: 3;
+}
+
+/* Bottom dock */
+#report-button {
+  bottom: 200px;
+  height: 200px;
+  position: absolute;
+  width: 200px;
+  z-index: 3;
```

Comment:
It's very rare to see so many usages of `z-index`. Can you omit them? If you just need the background to be behind everything, you can also just make the background `z-index: -1`, but even that might not be needed as long as your DOM order is right?

Reply:
Will fix this in a new PR.